conformity/fields/__init__.py

```diff
@@ -29,6 +29,9 @@
     IPv4Address,
     IPv6Address,
 )
+from conformity.fields.email import (  # noqa
```

Comment:
This is in the wrong place alphabetically. Please re-order.

Reply:
edited.